Sources/Web3Core/EthereumNetwork/Request/APIRequest+Methods.swift

```diff
@@ -51,10 +51,10 @@ extension APIRequest {
 
         /// This bit of code is purposed to work with literal types that comes in ``Response`` in hexString type.
         /// Currently it's just `Data` and any kind of Integers `(U)Int`, `Big(U)Int`.
-        if let LiteralType = Result.self as? LiteralInitiableFromString.Type {
+        if let LiteralType = Result.self as? LiteralInitableFromString.Type {
```

Comment:
Please undo this changes, as they affecting code, rather then comments and should be reviewed separately.